panel/pane/image.py

```diff
@@ -567,3 +567,36 @@ def _imgshape(cls, data):
                 w = int.from_bytes(b.read(2), 'little') + 1
                 h = int.from_bytes(b.read(2), 'little') + 1
         return int(w), int(h)
+
+class AVIF(ImageBase):
+    """
+    The `AVIF` pane embeds a .avif image file in a panel if
+    provided a local path, or will link to a remote image if provided
+    a URL.
+
+    Reference: https://panel.holoviz.org/reference/panes/AVIF.html
+
+    :Example:
+
+    >>> AVIF(
+    ...     'https://raw.githubusercontent.com/Kagami/avif.js/refs/heads/master/demo/Mexico.avif',
```

Comment:
I don't know how you found this image but please just use the same one as in the test.